src/main/java/cceclipseplugin/core/EclipseRequestManager.java

```diff
@@ -369,14 +370,14 @@ private void lookupProjectFilesAndPutMetadata(String path, ProjectMetadata meta)
 			if (m instanceof IFile) {
 				String path = ((IFile) m).getProjectRelativePath().toString();
 				if (ignoreFile.containsEntry(path)) {
-					System.out.println(String.format("File %s was ignored when scanning for files.", path));
+					Logger.getInstance().log(IStatus.INFO, String.format("File %s was ignored when scanning for files.", path));
 				} else {
 					files.add((IFile) m);
 				}
 			} else if (m instanceof IFolder) {
 				String path = ((IFolder) m).getProjectRelativePath().toString();
 				if (ignoreFile.containsEntry(path)) {
-					System.out.println(String.format("Folder %s was ignored when scanning for files.", path));
+					Logger.getInstance().log(IStatus.INFO, String.format("Folder %s was ignored when scanning for files.", path));
```

Comment:
There are 8 occurrences of `Logger.getInstance()` in this class. I think it should be called once and put in a field.